### PR TITLE
Track a metrics of number of registered watchers in apiserver

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go [hunk 1]
```diff
@@ -140,6 +141,9 @@ type WatchServer struct {
 // ServeHTTP serves a series of encoded events via HTTP with Transfer-Encoding: chunked
 // or over a websocket connection.
 func (s *WatchServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	metrics.RegisteredWatchers.WithLabelValues(s.Scope.Kind.Kind).Inc()
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go]: we don't want API group and version?

Comment (wojtek-t) [reply staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go]: Fair enough. Added.

**Comment by wojtek-t** [comment]
PTAL

**Pull request merged**